src/core/providers/qgsproviderregistry.cpp

```diff
@@ -411,6 +414,24 @@ void QgsProviderRegistry::init()
       }
     }
 
+    // now get vector tile file filters, if any
+    const QString fileVectorTileFilters = meta->filters( Qgis::FileFilterType::VectorTile );
+    if ( !fileVectorTileFilters.isEmpty() )
+    {
+      QgsDebugMsgLevel( "vector tile filters: " + fileVectorTileFilters, 2 );
```

Comment:
Maybe worth adjusting debug message so it matches preexisting ones?

Reply:
It's an exact match for the point cloud one just above! (I copy pasted all that code 😂)

Reply:
@nyalldawson , doh! I was looking at the ones further up. No worries then, let it be :)